docs/overview.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+parent: ClusterFuzzLite
+title: Overview
+has_children: false
+nav_order: 2
+permalink: /overview/
+---
+# Overview
```

Comment:
We should change clusterfuzzlite.md to link to this.

Reply:
Done. See related updates on clusterfuzzlite.md as well.

Reply:
THanks!